datetime/src/time.rs

```diff
@@ -81,11 +86,12 @@ fn time(args: Args) -> Result<(), Box<dyn std::error::Error>> {
 
     let mut child = Command::new(&args.utility)
         .args(args.arguments)
-        .stderr(Stdio::piped())
+        .stdout(Stdio::inherit())
+        .stderr(Stdio::inherit())
         .spawn()
-        .map_err(|_| format!("Command not found: {}", args.utility))?;
+        .map_err(|_| TimeError::ExecCommand(args.utility))?;
```

Comment:
Failure to spawn the command/util doesn't mean that the command doesn't exist. There could also be missing permissions, etc. This should map to either UtilError or UtilNotFound